### PR TITLE
Modern favicons

### PR DESCRIPTION
**Pull request by kaermorchen**
This PR adds modern favicons. It contains some icons:

1. `icon.svg` for modern browsers, the dark mode included
2. `favicon.ico` contains two 32px and 16px images for legacy browsers
3. `apple-touch-icon.png` for Apple devices
4. Also `icon-512`, `icon-192` and `manifest.webmanifest` for others

I have removed the old favicon.png file to reduce confusions

### CODE REVIEW

**Review by MinThaMie — APPROVED** [review]
Awesome :) thanks for the work!

**Review by MinThaMie — CHANGES_REQUESTED** [review]
@wifelette can you check if this is the color we want? It's not the same as the old one and also not the same icon as mentioned on the brading page. 

@kaermorchen thanks for this, where did you get the logo from :)

**Comment by kaermorchen** [comment]
> @kaermorchen thanks for this, where did you get the logo from :)

The logo is taken from [this directory](https://github.com/ember-learn/ember-website/tree/main/public/images/brand), don't remember which one.

The color is taken from [this pdf](https://github.com/ember-learn/ember-website/blob/main/public/pdfs/Ember-Brand-Guidelines.pdf), page 13.

I can change the colors, tell me to which one.

**Comment by wifelette** [comment]
This is lovely, thank you! That is indeed the correct page of the correct version of the guide. I can't recall if we updated the _assets_ at that time though, or just the guide.

TL;DR the orange color should be `#DB3122`, which is the _web accessible_ version of the Ember orange. If it is, we're good to go! 

If it isn't, you can either change the color and remake the png, or let me know that I need to get new assets and I shall :)

Thanks @kaermorchen for doing the work, @MinThaMie for being ever-vigilant, and @MelSumner for making sure we updated our Style Guide with the correct colors!

**Review by MinThaMie — APPROVED** [review]
Cool, thanks for the work!
